client/src/components/Shared/Button/Button.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { ButtonWrapper } from './Button.styled';
+
+const Button = ({ color, children, clickHandler, type }) => (
+	<ButtonWrapper type={type} color={color} onClick={clickHandler}>
+		{children}
+	</ButtonWrapper>
+);
+
+Button.propTypes = {
+	color: PropTypes.string,
+	type: PropTypes.string.isRequired, // simple | filled
```

Comment:
If the only options are 'simple' or 'filled', this could be `PropTypes.oneOf(['simple', 'filled']).isRequired`.

Reply:
Yeah, this is kinda still work in progress, I'm not really sure what kind of buttons we will need later, so I've added comment on the side just to not forget what I had in mind 😄